scipy/stats/tests/test_distributions.py

```diff
@@ -7109,6 +7109,16 @@ def test_burr_nan_mean_var_9544(self):
         assert_(np.isfinite(e3))
         assert_(np.isfinite(e4))
 
+    def test_burr_isf(self):
+        # reference values were computed via the reference distribution, e.g.
+        # mp.dps = 100
+        # Burr(c=c, d=d).sf(q).
```

Comment:
If the reference values were created with `sf`, is the `_ppf` method of the reference distribution needed?

Reply:
I tried with just the pdf and it returned an error related to not being able to find a root for the given tolerance. Then I just added the cdf and it resulted in zero division errors. Thus I tried added the ppf and it seemd to returned the desired correct values.

Reply:
That's fine; it's mostly about the comment. It says you used the `sf` method here. Did you use the `sf` method, or did you use the `ppf`/`isf` method?

Reply:
Fixed the comment. Thanks for pointing it out.